examples/plotting/interactive_IPF.py

```diff
@@ -0,0 +1,71 @@
+"""
+====================================
+Interactive IPF map with Euler angle
+====================================
+
+This example shows how to use
+:doc:`matplotlib event connections <matplotlib:users/explain/event_handling>`
+to add an interactive click function to the inverse pole figure (IPF) plot to
+retrieve the phase name and corresponding Euler angles from the location of
+click.
+"""
```

Comment:
We should add an info message to prevent people thinking that the orientation map is clickable. Similar to how [Matplotlib](https://matplotlib.org/stable/gallery/event_handling/close_event.html#sphx-glr-gallery-event-handling-close-event-py) informs about this.
```suggestion
to add an interactive click function to the inverse pole figure (IPF) map to
retrieve the phase name and corresponding Euler angles from the location of
click.

.. note::
    This example shows the interactive capabilities of Matplotlib, and this
    will not appear in the static documentation. Please run this code on your
    machine to see the interactivity.

    You can copy and paste individual parts, or download the entire example
    using the link at the bottom of the page.
"""
```

Reply:
Indeed, it make a lot more sense now.

Reply:
> The columns are the same in the Bruker CTF files I have access to.

Is there any difference in the metadata line in the header section?